packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -282,22 +348,63 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
 									Select {filesPlural}
 								</Button>
 							</Flex>
-							{showFileLists && (
+							{areFileListsVisible && (
 								<Stack gap={0.5}>
-									<Text color="muted">{fileSummaryText}</Text>
+									{Boolean(allRejections.length) && (
+										<SectionAlert
+											title="The following files could not be selected"
+											tone="error"
+											onClose={clearErrors}
+										>
+											<Text as="p">
+												These files did not meet upload requirements, were not
+												listed, and will not be uploaded.
```

Comment:
Can we soften this language at all?

```suggestion
												Please see a list of files and the reasons we are unable to upload them below.
```

Reply:
I changed it to this:

> Please review this list of files that we were unable to accept for the stated reasons: